framework/db/oci/Schema.php

```diff
@@ -57,7 +58,25 @@ public function releaseSavepoint($name)
      */
     public function quoteSimpleTableName($name)
     {
-        return strpos($name, '"') !== false ? $name : '"' . $name . '"';
+        if (strpos($name, '"') !== false) {
+            return $name;
+        } elseif ($this->db->slavePdo->getAttribute(PDO::ATTR_CASE) === PDO::CASE_LOWER) {
+            $name = strtoupper($name);
```

Comment:
It works as usual unless you define PDO::CASE_LOWER.
If you are using case-sensitive tables, you just can't use PDO::CASE_LOWER or you need to adress the table name with " as '"UserComment"' (just like in Oracle).
This is because, you can't expect the lowercase pdo behavior to work if there is a table "UserComment" and another table "USERCOMMENT".